t/10-npg_warehouse-loader-pacbio-run.t

```diff
@@ -324,6 +325,41 @@ subtest 'load_completed_run_on_instrument_deplexing_analysis3' => sub {
   is ($p->hifi_bases_percent, q[98.63], 'correct hifi reads for run TR539 well C1 barcode default');
 };
 
+
+subtest 'load_completed_run_on_instrument_deplexing_analysis4' => sub {
+  plan tests => 8;
+
+  my $pb_api = WTSI::NPG::HTS::PacBio::Sequel::APIClient->new(user_agent => $user_agent);
+
+  my @load_args = (dry_run       => '0',
+                   pb_api_client => $pb_api,
+                   mlwh_schema   => $wh_schema,
+                   run_uuid      => q[47cb98f3-b95d-4cec-95d2-1667afc118d1],
+                   hostname      => q[blah.sanger.ac.uk]);
+
+  my $loader   = npg_warehouse::loader::pacbio::run->new(@load_args);
+  my ($processed, $loaded, $errors) = $loader->load_run;
+
+  cmp_ok($loaded, '==', 1, "Loaded 1 completed post v13 run");
+  cmp_ok($errors, '==', 0, "Loaded 1 completed post v13 run with no errors");
+
+  my $rs = $wh_schema->resultset($RUN_WELL_TABLE_NAME)->search
+    ({pac_bio_run_name => 'TRACTION-RUN-1125', well_label => 'C1'});
+  is ($rs->count, 1, '1 loaded row found for run TR1125 well C1 in pac_bio_run_well_metrics');
+  my $r = $rs->next;
+
+  is ($r->ccs_execution_mode, q[OnInstrument], 'correct process type for run TR1125 well C1');
+  is ($r->hifi_num_reads, q[4100198], 'correct hifi reads for run TR1125 well C1');
+
+  is ($r->hifi_read_bases, q[43121860790],
+     'correct num of hifi bases for run TR1125 well C1');
+  is ($r->hifi_bases_in_barcoded_reads, q[43031425903],
+     'correct num of hifi bases in barcoded reads for run TR1125 well C1');
+  is ($r->plate_number, 1, 'correct plate number 1 for run TR1125 well C1');
```

Comment:
String comparison on an integer

Reply:
In this db output comparison context I prefer that.